lib/layout/list_view.dart

```diff
@@ -221,6 +221,7 @@ class ListViewState extends WidgetState<ListView>
           }
         }
         if (itemWidget != null) {
+          itemWidget = KeepAliveListItem(itemWidget: itemWidget);
```

Comment:
Flutter's documentation for `KeepAlive` states this - 

```
[ListView] and
/// [GridView], have an `addAutomaticKeepAlives` feature, which is enabled by
/// default, and which causes [AutomaticKeepAlive] widgets to be inserted around
/// each child, causing [KeepAlive] widgets to be automatically added and
/// configured in response to [KeepAliveNotification]s.
```
We use `SilverList` and that is already setting `addAutomaticKeepAlives` to true so why do we need to do this again here?